### ISSUE TITLE
rake_support is wack: should require SolrWrapper, FcrepoWrapper, be a Module (or helper, or tasks)

### ISSUE THREAD

**Issue by atz** [samvera/active_fedora#1179]
The file is just a bunch of methods, not a Class or Module.  

It invokes `SolrWrapper` and `FcrepoWrapper` without requiring those gems.

https://github.com/projecthydra/active_fedora/blob/v11.0.1/lib/active_fedora/rake_support.rb#L17-L32

It also includes a boolean conditional based on a ENV variable **string** value.  
https://github.com/projecthydra/active_fedora/blob/v11.0.1/lib/active_fedora/rake_support.rb#L8-L10

Here's some IRB showing why that is broken:
```ruby
> puts "foo" if 'false'
(irb):1: warning: string literal in condition
foo
 => nil 
> puts "foo" if 'true'
(irb):2: warning: string literal in condition
foo
```

Basically, `ENV["#{environment}_SERVER_STARTED"] = 'false'` does not affect the execution in the way the author expected.  For the purposes of the conditional, it is the same as doing nothing.

**Comment by atz** [comment]
And what does `private` mean in a non-class non-module position?
https://github.com/samvera/active_fedora/blob/v11.0.1/lib/active_fedora/rake_support.rb#L36

Is the author trying to do OO or not?

**Comment by jcoyne** [comment]
The current code prevents a dependency on solr_wrapper and fcrepo_wrapper in production

**Comment by jcoyne** [comment]
We never set 'false' its working because it's effectively doing a nil check
